air-quality-ui/system_tests/ui/summary_page.spec.ts

```diff
@@ -1,75 +1,807 @@
+test.describe('Table Data: Largest Deviation', () => {
+  test('Verify table shows pollutant data for the timestamp that has the largest deviation - forecast AQI 6, measurement AQI 3', async ({
+    page,
     vairifySummaryPage,
   }) => {
-    await vairifySummaryPage.getColumnHeaderAndText('AQI Level', 'AQI Level')
-    await vairifySummaryPage.getColumnHeaderAndText(
-      'PM 2.5 (µg/m³)',
-      'PM 2.5 (µg/m³)',
-    )
-    await vairifySummaryPage.getColumnHeaderAndText(
-      'PM 10 (µg/m³)',
-      'PM 10 (µg/m³)',
-    )
-    await vairifySummaryPage.scrollToRightmostPosition()
-    await vairifySummaryPage.page.waitForTimeout(1000)
+    const forecastLondonValidTimeArray: object[] = [
+      createForecastAPIResponseData(),
+      createForecastAPIResponseData({
+        valid_time: '2024-07-08T03:00:00Z',
+      }),
+      createForecastAPIResponseData({
+        valid_time: '2024-07-08T06:00:00Z',
+        overall_aqi_level: 6,
+        no2: { aqi_level: 6, value: 900 },
+        o3: { aqi_level: 6, value: 400 },
+        pm2_5: { aqi_level: 6, value: 80 },
+        pm10: { aqi_level: 6, value: 160 },
+        so2: { aqi_level: 6, value: 800 },
+      }),
+      createForecastAPIResponseData({
+        valid_time: '2024-07-08T09:00:00Z',
+        overall_aqi_level: 5,
+        no2: { aqi_level: 5, value: 240 },
+        o3: { aqi_level: 5, value: 370 },
+        pm2_5: { aqi_level: 5, value: 60 },
+        pm10: { aqi_level: 5, value: 120 },
+        so2: { aqi_level: 5, value: 510 },
+      }),
+      createForecastAPIResponseData({
+        valid_time: '2024-07-08T12:00:00Z',
+        overall_aqi_level: 4,
+        no2: { aqi_level: 4, value: 125 },
+        o3: { aqi_level: 4, value: 135 },
+        pm2_5: { aqi_level: 4, value: 26 },
+        pm10: { aqi_level: 4, value: 99 },
+        so2: { aqi_level: 4, value: 205 },
+      }),
+    ]
 
-    await vairifySummaryPage.getColumnHeaderAndText(
-      'Nitrogen Dioxide (µg/m³)',
-      'Nitrogen Dioxide (µg/m³)',
-    )
-    await vairifySummaryPage.getColumnHeaderAndText(
-      'Ozone (µg/m³)',
-      'Ozone (µg/m³)',
+    const measurementsLondonArray: object[] = [
+      createMeasurementSummaryAPIResponseData(),
+      createMeasurementSummaryAPIResponseData({
+        measurement_base_time: '2024-07-08T03:00:00Z',
+      }),
+      createMeasurementSummaryAPIResponseData({
+        measurement_base_time: '2024-07-08T06:00:00Z',
+      }),
+      createMeasurementSummaryAPIResponseData({
+        measurement_base_time: '2024-07-08T09:00:00Z',
+      }),
+      createMeasurementSummaryAPIResponseData({
+        measurement_base_time: '2024-07-08T012:00:00Z',
+      }),
+    ]
+
+    await vairifySummaryPage.setupPageWithMockData(
+      forecastLondonValidTimeArray,
+      measurementsLondonArray,
     )
-    await vairifySummaryPage.getColumnHeaderAndText(
-      'Sulphur Dioxide (µg/m³)',
-      'Sulphur Dioxide (µg/m³)',
+
+    const expectedTableContents: string[][] = [
```

Comment:
How come we aren't doing await vairifySummaryPage.assertGridValues(expectedData) like we do in the test above?